functions/Find-PasswordStatePassword.ps1

```diff
@@ -104,7 +107,9 @@ Function Find-PasswordStatePassword {
         [Parameter(ParameterSetName = 'Specific', ValueFromPipelineByPropertyName, Position = 17)][string]$GenericField8,
         [Parameter(ParameterSetName = 'Specific', ValueFromPipelineByPropertyName, Position = 18)][string]$GenericField9,
         [Parameter(ParameterSetName = 'Specific', ValueFromPipelineByPropertyName, Position = 19)][string]$GenericField10,
-        [parameter(ValueFromPipelineByPropertyName, Position = 20)][string]$Reason
+        [Parameter(ParameterSetName = 'General', ValueFromPipelineByPropertyName, Position = 1)]
```

Comment:
Is this line a mistake? There is no parameter on the parameter block.

Reply:
That should be correct

Reply:
Not sure i understand? General only has one parameter which is search to search everything?
Which parameter are we defining twice?

Reply:
https://mikefrobbins.com/2018/02/22/adding-multiple-parameter-sets-to-a-powershell-function/

I need the PasswordListID in the General ParameterSet and in the Specific ParameterSet

Reply:
Oh i understand now. Because you want the position to be both 20 and 1 you are specifying it twice.

Makes sense.